pentagon/__init__.py

```diff
@@ -13,13 +10,17 @@
 import re
 import boto3
 import sys
+import pentagon.component.vpc as aws_vpc
+from pentagon.helpers import render_template
+
```

Comment:
Could these be moved down to the other pentagon import below or do they need to be imported before other 3rd party libraries? Keeping similar imports grouped together helps keep code organized.

Reply:
Sure